RoddyCore/test/src/de/dkfz/roddy/execution/io/LocalExecutionServiceSpec.groovy

```diff
@@ -16,52 +16,55 @@ class LocalExecutionServiceSpec extends Specification {
 
     LocalExecutionService es = ExecutionService.instance
 
-    def "execute('echo')"() {
+    def "execute synchronously and succeed with captured stdout and non!-ignored stderr"() {
```

Comment:
JPA and I checked all the names, thanks for improving them!
One seems to have run into a copy-paste error (see comment below), but otherwise good :+1: